### ISSUE TITLE
IE7.js and IE8.js are makin Internet Explorer 7 regress

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#50]
```
IE7.js and IE8.js make IE7 regress

If you try with one of the two scripts ie7.js or ie8.js linked on your web
page, to style an element by pointing its attributes having a value, you
will notice that it doesn'work in IE7 but works in IE6.

For example :

css :

img[alt="computer"] {border:2px solid red;}

html :

<img src="images/pc.jpg" alt="computer" width="100" height="100" />

Will render an image with a red border in IE6 but not IE7.

The same with img[alt=""] (However, it works with img[alt] without any values).

It works the same with the attribute title and src.

```

Original issue reported on code.google.com by `eric.eri...@gmail.com` on 6 Mar 2008 at 2:32

Attachments:
- [ie7ie8-alt-img.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-50/comment-0/ie7ie8-alt-img.html)

**Comment by GoogleCodeExporter** [comment]
```
As a workaround, why not include the script conditionally for IE <7 only?

<!--[if lt IE 7]>
<script ... >
<![endif]-->
```

Original comment by `wdha...@gmail.com` on 16 Apr 2008 at 10:26

**Comment by GoogleCodeExporter** [comment]
```
IE8.js also has the regression...
```

Original comment by `pierre6...@gmail.com` on 16 Apr 2008 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 1:31
- Changed state: **Fixed**
